### PR TITLE
Fea/iss 5/code linter

### PR DESCRIPTION
**Pull request by chrishalbert**
# Description of Work

[Add code linter](https://github.com/hop-along-polly/always-on/issues/5)

This work adds the linter `black` as a dev dependency to the `requirements-dev.txt`, along with some `Make` targets to run the lint checks locally AND even do some of the fixes! This also fulfills the AC's listed in the referenced issue above.

image

# Type of Change
I'm split on whether this is a Chore or a Feature. I feel like a Feature refers more to the application, where as this is code/repo overhead.
- [ ] Bugfix (A non-breaking change that fixes an issue)
- [X] Feature (A non-breaking change that adds new functionality)
- [ ] Chore (A non-breaking change that adds no net new functionality, tests, refactoring, docs etc.)
- [ ] Breaking Change (a fix or feature that is not backwards compatible with existing functionality)

# Testing Steps

This should run once the github actions are in place.  To illustrate the functionality, I'll push a failing commit to illustrate the ❌, as well as the subsequent ✅ 

# Definition of Done Checklist

- [x] I have only included commits that apply to the referenced Github Issue.
- [x] I have performed a self-review of my own code.
- [x] I have taken the time to refactor the code to ensure testability, and maintainability.
- [x] I have made corresponding changes to the documentation (if needed).
- [x] I have added tests that prove my fix/feature is working per requirements.
- [x] I have provided sufficient code comments so any developer can support my code.

### CODE REVIEW

File: alwayson/app.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 # Setup AlwaysON System Routes
 # The sys_router for is for Always ON System Routes
-sys_router = APIRouter(prefix='/v1')
+sys_router = APIRouter(prefix="/v1")
```
Comment (hop-along-polly) [inline alwayson/app.py:13]: oooh I think we're going to have a philosophical difference here. I like the single quotes until I need double quotes.
(i.e. `"This isn't doable with single quotes"`

Comment (hop-along-polly) [reply alwayson/app.py:13]: That being said I'm fine using Black but I'm surprised it didn't catch more instances where I was using single-quotes.

**Review by hop-along-polly — APPROVED** [review]

File: alwayson/app.py (hunk 1 above)
Comment (chrishalbert) [reply alwayson/app.py:13]: Nice catch, I'll see if there's something I missed

Comment (chrishalbert) [reply alwayson/app.py:13]: I just did a search and I'm only seeing double quotes in the `.py` files.

Comment (hop-along-polly) [reply alwayson/app.py:13]: Yea all looks good. Must have been a glitch as to why I was only seeing this change 🤷

**Review by hop-along-polly — APPROVED** [review]
This looks good to me. The only concern I had is that Black is changing some white space. I'm hoping that's just increasing the # of spaces used for indentation, rather than tabs. I'm one of those people that hate tabs.

**Pull request merged**